transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringBufferRingRecvAllocator.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2025 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.uring;
+
+import io.netty.buffer.ByteBuf;
+
+/**
+ * Allocator that is responsible to allocate buffers for a buffer ring.
+ */
+public interface IoUringBufferRingRecvAllocator {
```

Comment:
Is this a good name ?

Reply:
Myabe just `IoUringBufferRingAllocator` so it's clearly part of the buffer ring machinery without loading more expectations upon it than the buffer ring itself has.